spec.md

```diff
@@ -3697,6 +3746,9 @@ statement =
   | "when" , expr , ":" [ info ] , newline ,
     indent , statement, { statement } , dedent ,
     [ "else" , ":" , indent , statement, { statement } , dedent ]
+  | "match" , expr , ":" , [ info ] , newline ,
+    [ indent , { id , [ "(" , id , ")" ] , ":" , newline , 
+    [ indent , { statement } , dedent ] } , dedent ]
```

Comment:
Nit: Increasing the indentation for the second indent would help clarify what is going on here. Also, can't this repeat and isn't optional? You would likely need to define a separate `match_statement` and then show that that repeats:

Reply:
I was trying to make the "case"s entirely optional in the grammar.  It was kind of tricky to do without accidentally allowing each "case" can have a different level of indentation.
e.g. the following is illegal because of the change in indentation, but having empty statement lists is fine:

```
match x:
  a:

   b:
```

There are two `{ }` operators nested, but the outer one was really hard to see the way I had flowed the text.  This might be breaking it up a bit too much but might help read the pattern:
```
  | "match" , expr , ":" , [ info ] , newline ,
    [ indent ,
      { id , [ "(" , id , ")" ] , ":" , newline , 
        [ indent ,
          { statement } ,
        dedent ]
      } ,
    dedent ]
```